src/api/procedures/setTransferRestrictions.ts

```diff
@@ -18,88 +25,128 @@ import {
   TxTag,
   TxTags,
 } from '~/types';
-import { ProcedureAuthorization, StatisticsOpType } from '~/types/internal';
+import { ProcedureAuthorization } from '~/types/internal';
 import {
   complianceConditionsToBtreeSet,
-  scopeIdsToBtreeSetIdentityId,
-  statisticsOpTypeToStatOpType,
-  stringToIdentityId,
+  identitiesToBtreeSet,
+  identityIdToString,
+  meshClaimTypeToClaimType,
   stringToTickerKey,
   toExemptKey,
   transferRestrictionToPolymeshTransferCondition,
+  transferRestrictionTypeToStatOpType,
   u32ToBigNumber,
 } from '~/utils/conversion';
 import {
   assertStatIsSet,
   checkTxType,
   compareTransferRestrictionToInput,
-  getExemptedIds,
   neededStatTypeForRestrictionInput,
 } from '~/utils/internal';
 
 /**
  * @hidden
  */
+
+type ClaimKey = keyof typeof ClaimType | 'None';
```

Comment:
Shouldn't this be limited to the 3 claim types that are allowed in transfer restrictions?

Reply:
On chain it's an option type, with `None` being for `Count` and `Percentage`.

There might be a better way than having a magic string of 'None' though

Reply:
What I meant was having `type ClaimKey = ClaimType.Accredited | ClaimType.Affiliate | ClaimType.Jurisdiction | 'None'`